client/src/actions/actionsUser/usersActions.js

```diff
@@ -170,5 +170,82 @@ export const fetchUser = (user_id) => {
       .catch(error => {
         dispatch(fetchUserFailure(error.message))
       })
+    }
   }
+
+  export const addAdmin = (name) => {
+  return function(dispatch, getState) {
+      dispatch(fetchUserRequest())
+      let base64 = require('base-64');
+      const username = localStorage.getItem("username");
+      const password = localStorage.getItem("password");
+      const token= base64.encode(username+":"+password);
+       axios.put('http://localhost:55555/user/activateAdmin',{
+        username:name
+       },{
+        headers: {'Authorization' : 'Basic '+ token},
+      })
+        .then( res =>{
+          console.log(res.data);
+           dispatch(fetchUserSucess(res.data));
+         alert('Ahora '+name+' es Admin');
+        })
+        .catch(error => {
+          console.log(error)
+         dispatch(fetchUserFailure(error.message));
+         alert('Hubo un problema con hacer admin a : '+name+ ' o este usuario no existe');
+        })
+    }
 }
+
+  export const disableAcc = (user, pass) =>{
+  return function(dispatch) {
+    dispatch(fetchUserRequest())
+     let base64 = require('base-64');
+      const username = localStorage.getItem("username");
+      const password = localStorage.getItem("password");
+      const token= base64.encode(username+":"+password); 
+     axios.put('http://localhost:55555/user/disableAccount',{
```

Comment:
instead `localhost` could you  to use `process.env.REACT_APP_API_HOST` and you can configure it properly in your `client/.env` file?